test/CMakeLists.txt

```diff
@@ -23,6 +23,9 @@
  #
  #######################################################################################################################
 
+# This CMakeLists.txt simply forwards to the real one in llpc/test, and is present only
+# to cater for an old build script that still looks for it in the old location.
+
```

Comment:
This comment is not true anymore and was removed on purpose.

Reply:
should I remove it? if so , we need to open a PR and port it to internal branch

Reply:
Yes, that would be nice. Will you do that or should I open a PR?